includes/admin/reporting/export/export-actions.php

```diff
@@ -177,6 +177,29 @@ function edd_include_file_downloads_batch_processer( $class ) {
 }
 
 /**
+ * Register the sales batch exporter
+ *
+ * @since  2.7
+ */
+function edd_register_sales_export_batch_export() {
+	add_action( 'edd_batch_export_class_include', 'edd_include_sales_export_batch_processer', 10, 1 );
+}
+add_action( 'edd_register_batch_exporter', 'edd_register_sales_export_batch_export', 10 );
```

Comment:
Should we drop this to a lower priority like `9` or something? That way no matter what the load order is of the plugins on the site, we're running before default priority?